addons/phone_validation/tests/test_phonenumbers_patch.py

```diff
@@ -28,5 +28,39 @@ def test_region_CI_monkey_patch(self):
             self.assertEqual(parsed_phonenumber_2.national_number, 22522586, "The national part of the phonenumber should be 22522586")
             self.assertEqual(parsed_phonenumber_2.country_code, 225, "The country code of Ivory Coast is 225")
         else:
-            self.assertFalse(hasattr(phonenumbers_patch, '_local_load_region'),
-                "The code should not be monkey patched with phonenumbers > 8.12.32.")
+            self.assertNotEqual(
+                phonenumbers.PhoneMetadata._region_available['CI'],
+                phonenumbers_patch._local_load_region,
+                "The code should not be monkey patched with phonenumbers > 8.12.32.",
+            )
+
+    def test_region_MU_monkey_patch(self):
+        """Makes sure that patch for Mauritius phone numbers work"""
+        if not phonenumbers:
+            self.skipTest('Cannot test without phonenumbers module installed.')
+        # MONKEY PATCHING phonemetadata of Mauritius if phonenumbers is too old
+        if parse_version(phonenumbers.__version__) < parse_version('8.12.13'):
+            # check that _local_load_region is set to `odoo.addons.phone_validation.lib.phonenumbers_patch._local_load_region`
+            # check that you can load a new Mauritius phone number without error
+            for test_number, country_region, exp_national_number, exp_country_code in [
```

Comment:
Isn't this test also valid for other versions ?  I mean we can also check it works for newer versions, even with standard code :)